components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -338,6 +339,38 @@ void RewardsServiceImpl::Init() {
   ledger_->Initialize();
 }
 
+void RewardsServiceImpl::MaybeShowBackupNotification() {
+  PrefService* pref_service = profile_->GetPrefs();
+  bool user_has_funded = pref_service->GetBoolean(kRewardsUserHasFunded);
+  bool backup_succeeded = pref_service->GetBoolean(kRewardsBackupSucceeded);
+  if (user_has_funded && !backup_succeeded) {
+    base::Time now = base::Time::Now();
+    base::Time boot_timestamp = pref_service->GetTime(kRewardsBootTimestamp);
```

Comment:
we have boot time in native ledger. I think we should use that one. Currently we don't have it exposed, but we can add getter for it

Reply:
Fixed.